Fixes <https://issues.guix.gnu.org/65177>.

Run 'vgchange' directly instead of attempting to run it via 'systemd-run' as
the udev rules included in lvm2 >= 2.03.14 do.

diff --git a/udev/69-dm-lvm.rules.in b/udev/69-dm-lvm.rules.in
index ff15681..f9e5b70 100644
--- a/udev/69-dm-lvm.rules.in
+++ b/udev/69-dm-lvm.rules.in
@@ -75,8 +75,8 @@ LABEL="lvm_scan"
 # TODO: adjust the output of vgchange -aay so that
 # it's better suited to appearing in the journal.
 
-IMPORT{program}="(LVM_EXEC)/lvm pvscan --cache --listvg --checkcomplete --vgonline --autoactivation event --udevoutput --journal=output $env{DEVNAME}"
-ENV{LVM_VG_NAME_COMPLETE}=="?*", RUN+="(SYSTEMDRUN) --no-block --property DefaultDependencies=no --unit lvm-activate-$env{LVM_VG_NAME_COMPLETE} (LVM_EXEC)/lvm vgchange -aay --autoactivation event $env{LVM_VG_NAME_COMPLETE}"
+IMPORT{program}="(LVM_EXEC)/lvm pvscan --cache --listvg
+ENV{LVM_VG_NAME_COMPLETE}=="?*", RUN+="(SYSTEMDRUN) --no-block lvm-activate-$env{LVM_VG_NAME_COMPLETE} (LVM_EXEC)/lvm vgchange -aay
 GOTO="lvm_end"
 
 LABEL="lvm_end"
